.changeset/catalog-compose-my-techdocs.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-catalog': patch
```

Comment:
Bit torn on whether we can actually call this patch. It's definitely gonna break behaviour, but not compilation or runtime. Perhaps we can mark it a bit more visibly with a **NOTE** or mth?

Reply:
We had a similar pause when creating this, yeah.  Are you thinking literally just a bold **Note** in the changeset text @Rugvip?